ws-nextjs-app/pages/api/local/[service]/[pageType]/[id]/[[...variant]].api.ts

```diff
@@ -0,0 +1,56 @@
+import fs from 'node:fs/promises';
+import path from 'path';
+import type { NextApiRequest, NextApiResponse } from 'next';
+
+type RequestPathParts = {
+  service: string;
+  pageType: string;
+  id: string;
+  variant?: string[];
```

Comment:
You may be able to use the `Services`, `Variants` and `PageTypes` types here to type-check this, but just a minor nitpick!

Reply:
Thanks for pointing this out. I've updated `RequestPathParts` to use `Services`, `Variants` and `PageTypes`.